### ISSUE TITLE
[Preview-MAS Violation]: Screen reader Is not narrating the names of the non-selected tab in cursor mode.

### ISSUE THREAD

**Issue by NavyaGoberu** [nvaccess/nvda#9722]
**User Experience:**
Users who rely on screen readers will be affected if Screen reader Is not narrating the names of the non-selected tab in cursor mode.

**Test Environment:**
OS: Win 10
Browser: EDGE, Chrome, pixel2
URL:  https://www.microsoft.com/en-us/enterprise/retail/empower-your-people 
Screen readers: Narrator, Nvda, Talkback

**Repro Steps:**
Step 1: Open the above URL using Edge latest browser
Step 2: Navigate to the tab elements under the heading “How Microsoft is empowering retail employees”.
Step 3: Verify the narration of the screen reader when the focus is on the selected tab item.
 
**Actual Result:**
Screen reader Is not narrating the names of the non-selected tab in cursor mode.

**Expected Result:**
Screen reader should narrate the names of the non-selected tab in cursor mode.

Note: Please find the Reference URL's below.
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

**MAS Reference:**
MAS 4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by josephsl** [comment]
Hi, when you say Edge, are you talking about the current (EdgeHTML) or upcoming (Chromium) version? Also, does this problem occur with Firefox and Internet Explorer? Thanks.

**Comment by NavyaGoberu** [comment]
Hi,

It is EdgeHTML.
This issue doesn’t repro in Firefox and Internet Explorer

Thanks,
Navya.

**Comment by Adriani90** [comment]
This works as expected in Edge Cromium 81, Crome 81 and Firefox 75 with NVDA Version: alpha-20015,56a62097.
Closing as works for me since Edge Cromium will be distributed to most users in 20 H1 update of Windows 10.

**Comment by Adriani90** [comment]
Adding appropriate label since it is probably still reproducible in Edge Spartan and UIA related environment.
